server/sonar-db-dao/src/main/java/org/sonar/db/DbClient.java

```diff
@@ -71,6 +72,7 @@
 
   private final Database database;
   private final MyBatis myBatis;
+  private final DBSessions DBSessions;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "DBSessions" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00116)